host_io/playbooks/playbooks.json

```
@@ -0,0 +1,422 @@
+{
+  "type": "workflow_collections",
+  "data": [
+    {
+      "uuid": "897c3684-ec8b-4ad2-981f-031dccd010b7",
+      "@type": "WorkflowCollection",
+      "name": "Sample - host.io - 1.0.0",
+      "description": "host.io helps to get comprehensive domain name data, uncover new domains and the relationships between them, get DNS details, scraped website content, outbound links, backlinks, and other hosting details for any domain. This connector facilitates automated operation related to various domains.",
+      "visible": true,
+      "image": null,
+      "recordTags": [
+        "Host_io",
+        "host_io"
+      ],
```

```suggestion
      "recordTags": [
        "host_io"
      ],
```
Please update as above I will fix this in PB script.